### PR TITLE
Relpos entities fix

### PR DESCRIPTION
**Pull request by Bam4d**

### CODE REVIEW

**Comment by Bam4d** [comment]
Super simple exclusion list for entities such as global entities and other entities that do not have positional information to be excluded from the self-attention calculations.

Running some experiments with this now also so will see if they make much difference in clusters envs

File: rogue_net/rogue_net/relpos_encoding.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class RelposEncodingConfig:
     obs_space: ObsSpace
     d_head: int
     per_entity_values: bool = True
+    exclude_entities: Optional[List[str]] = None
```
Comment (cswinter) [inline rogue_net/rogue_net/relpos_encoding.py:17]: Can we just make this a `List[str]` with default of `[]` (using dataclasses.field)? (Really it should be `Set[str]` but don't know if that works with our current json loader).

**Review by cswinter — APPROVED** [review]
LGTM just make `exclude_entities` non-optional

**Comment by Bam4d** [comment]
> An alternative would be to automatically ignore all entities that don't have the specified position features. Would be more ergonomic but could also be surprising/error-prone, so might still want to have a toggle that enables this.

I feel like having this explicit will be far better, or maybe a warning on construction at the very least. Having things "magically work" by assuming the intentions of users can be crazy frustrating for anyone using it.

**Pull request merged**